spec/components/all_components_spec.rb

```diff
@@ -0,0 +1,48 @@
+RSpec.describe "AllComponents", type: :controller do
```

Comment:
I refined my thinking on this a bit doing the components in government-frontend, so you're right it definitely shouldn't be a controller test. It might be that a view test is best, but I'll check whether taking the type out makes no difference, and if so I'll do that.

Reply:
Done.